dips/dip-10.md

```diff
@@ -101,130 +110,87 @@ struct DiemIdDomainEvent {
+|-------	    |------	     |-----------	|-------------	           |
+| _ObjectType   | str        | Y | Fixed value: `CommandResponseObject`|
+| status       | str | Y | Either `success` or `failure`. |
+| result       | Result object | Y | The Result obejct of response. |
+| cid           | str         | Y            | A unique identifier for the Command. Should be a UUID according to [RFC4122](https://tools.ietf.org/html/rfc4122) with "-"'s included. |
 
-* `Payment_Reference_ID` must be unique for each payment, it must not be reused in any way to avoid leaking privacy information on-chain.
+**CommandResultObject:**
 
-* `Payment_Reference_ID` returned by this endpoint is not signed for KYC purposes. This means that payment above KYC threshold can not be submitted right away using the provided Payment_Reference_ID. Instead, sender VASP needs to proceed to KYC endpoint to negotiate the signed Reference ID.
+| Field 	    | Type 	     | Required? 	| Description 	           |
+|-------	    |------	     |-----------	|-------------	           |
+| _ObjectType   | str        | Y | Fixed value: `ReferenceIDCommandResponse`|
+| recipient_address       | str | Y | Receiver's on-chain address |
 
-* KYC endpoint API will be updated to accept `Payment_Reference_ID` returned by the Diem ID endpoint instead of address/subaddress.
-
-## User Object
-User Object defines reach information about the user.
-
-All fields, except for `diem_id`, in this object are optional - VASP can choose to share either of those fields or none.
-
-Additionally, all fields in this object are informational - **they must not be used for KYC and only intended for UI purposes**.
-```json
-{
-   "diem_id": "<Diem_ID>",
-   "display_name": "<Display name>",
-   "profile_picture": "<profile_picture_uri>"
+If the amount is below the travel rule limit, the sending VASP can send a p2p transaction with PaymentMetadata and the `reference_id` VASPs agreed on to settle the transaction. 
+```
+enum PaymentMetadata {
+    PaymentMetadataV0(ReferenceId),
 }
+type ReferenceId = [u8, 16];
 ```
 
-| Field 	      | Type  | Required? 	| Description 	           |
-|-------	      |------ |-----------	|-------------	           |
-|diem_id         |str    | Y           | Diem ID of the user |
-|display_name     |str    | N           | Display name of the user. Usually a full name |
-|profile_picture  |str    | N           | Profile picture URL(see below) |
-
-If a user does not exist, VASP can choose to return an HTTP error, or the empty User Object, depending on it’s privacy choices.
-
-## Profile Picture URI
-Profile picture URI is a URI that represents a profile picture of a user. Wallet app must support two possible URI schemas for profile picture.
+If the amount exceeds the travel rule limit, VASPs should trigger an off-chain protocol for KYC exchange. The same `reference_id` must be used to perform a TR as described in [DIP-1](https://github.com/diem/dip/blob/master/dips/dip-1.md).
```

Comment:
current dip-1's schema uses subaddress hence there is a gap. I suggest at this point we do not talk about Travel Rule in dip-10. We can have an extension dip for it when we are sure what we want for Travel Rule